### DISCUSSION TITLE
Styling entities in ERDs

### DISCUSSION THREAD

**Discussion by data-igor** [mermaid-js/mermaid#3874] [Ideas]
Currently as per this [discussion](https://github.com/orgs/mermaid-js/discussions/3838#discussion-4586893) it seems to be impossible to style individual entities. The problem seems to be based on the whole diagram using a single class.

At the same time other diagrams have this capability.
Adding this functionality could improve the readability of ERDs.

One use case would be to color the initial table, or color them by the schema e.g. raw/transformation/presentation.

**Comment by dlecan** [comment]
Please, I need this.
I can’t even use special chars in entities name (* or _) to highlight information

Any known workaround?
